unity-ggjj/Assets/Scripts/UI/PageBar.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PageBar : MonoBehaviour
+{
+    [Tooltip("An object to represent a page dot")]
+    [SerializeField] private Image _pageDot;
+    
+    private readonly List<Image> _pageDots = new List<Image>();
+
+    /// <summary>
+    /// Creates dots used to represent each page in the menu
```

Comment:
Nit: I think updating this comment to include the removal of dots when there's less pages than currently instantiated images:
```suggestion
    /// Creates or deletes images instances inside <see cref="_pageDots"/> until the length matches <see cref="pageCount"/>
```